File: backend/services/post-rust-actix-web-fdm/src/common/api/invalid_page.rs
use crate::common::utils::error::*;

impl super::super::Page {
    pub(in crate::common) fn invalid_page() -> ErrorBody {
        ErrorBody {
            error: ErrBody {
                error: "INVALID_PAGE".into(),
                reason: "page is invalid".to_string(),
                message: "page is invalid".to_string(),
            },
        }
    }
}

File: backend/services/post-ktor-mongo-fdm/src/main/kotlin/me/jason5lee/post_ktor_mongo_fdm/common/api/InvalidPage.kt
package me.jason5lee.post_ktor_mongo_fdm.common.api

import me.jason5lee.post_ktor_mongo_fdm.common.utils.Err
import me.jason5lee.post_ktor_mongo_fdm.common.utils.FailureBody

object InvalidPage {
    val invalidPage: FailureBody = FailureBody(
        error = Err(
            error = "INVALID_PAGE",
            reason = "page is invalid",
            message = "page is invalid",
        )
    )
}
